iFixit/react-commerce
# Add product page lifetime warranty section

closes #782 

## QA

1. Visit the Vercel preview
2. Go to a [product page](https://react-commerce-git-add-product-page-lifetime-warr-d494c2-ifixit.vercel.app/Products/pro-tech-toolkit)
3. If the selected variant has lifetime guarantee it should display a lifetime guarantee section at the bottom of the page
4. Verify that the section is implemented according to mockup and is responsive
